packages/ng/src/app/date-range-picker/date-range-modal/date-range-modal.component.ts

```diff
@@ -18,12 +18,12 @@ export class DateRangeModalComponent implements OnInit {
 
 	constructor (
 		@Inject(MAT_DIALOG_DATA) public data: IDateRange,
-		public dateAdapter: MomentDateAdapter,
+		// public dateAdapter: MomentDateAdapter,
 		public translate: LuTranslateService,
 		public dialogRef: MatDialogRef<any>
 	) {
-		this.locale = translate.getCurrentLang();
-		dateAdapter.setLocale(this.locale);
+		// this.locale = translate.getCurrentLang();
+		// dateAdapter.setLocale(this.locale);
```

Comment:
Does it mean we don't handle locale anymore ? (if so, we need to at least set it as a TODO)